client/landing/stepper/declarative-flow/internals/steps-repository/importer-migrate-message/index.tsx

```diff
@@ -120,8 +126,8 @@ const ImporterMigrateMessage: Step = ( { navigation } ) => {
 		text: __( `We'll help you switch your domain over after the migration is complete.` ),
 	} );
 
-	const title = hasEnTranslation( "We'll take it from here!" )
-		? __( "We'll take it from here!" )
+	const title = hasEnTranslation( "We've received your migration request" )
```

Comment:
The translation seems already available so that we can remove the `hasEnTranslation` verification.

https://translate.wordpress.com/projects/wpcom/-all-translated/973725/

Reply:
Done!